README.md

```diff
@@ -50,9 +55,9 @@ ________________________________________________________________________________
 ## Overview
 
 ### What is Scorecards?
-
-We created Scorecards to give consumers of open-source projects an easy way to
-judge whether their dependencies are safe.
+We created Scorecards to give maintainers of open-source projects a way to
```

Comment:
Small tweak: "We created Scorecards to help open source maintainers improve their security best practices and to help open source consumers judge whether their dependencies are safe."

Reply:
Done.